_data/events1.yml

```diff
@@ -0,0 +1,78 @@
+- category: Online Event

```

Comment:
Have you tried update the event.yml with this format to confirm that is valid yml?

Reply:
Yes. YAML format etc is fine and I could override, update the existing events.yml file.